### PR TITLE
Add "My vote" data to merge requests

### PR DESCRIPTION
**Pull request by TomBerriot**
Add personal vote state according to https://github.com/TheGrowingPlant/gitlab-reviewer/issues/15

68747470733a2f2f7075752e73682f445737487a2f333137653135666338612e706e67

### CODE REVIEW

File: src/components/MergeRequest/MergeRequestItem.tsx [hunk 1]
```diff
@@ -17,34 +20,52 @@ class MergeRequestItem extends Component<Props> {
...
+                  className="border-2 border-gray-500 h-10 w-10"
+                />
+              </td>
+              <td className="w-24 text-center">
+                {haveIUpVoted ? (
+                  <UpvoteIcon size={'lg'} />
+                ) : haveIDownVoted ? (
+                  <DownvoteIcon size={'lg'} />
+                ) : (
+                  <QuestionIcon size={'lg'} />
```
Comment (notFloran) [inline src/components/MergeRequest/MergeRequestItem.tsx]: I dont know if "question" is the right icon for that

Comment (notFloran) [reply src/components/MergeRequest/MergeRequestItem.tsx]: @TomBerriot @nclsHart

Comment (nclsHart) [reply src/components/MergeRequest/MergeRequestItem.tsx]: I could not find something more appropriate. Maybe leaving a blank space would simply be the best option.

**Comment by TomBerriot** [comment]
@nclsHart I don't see the value in supporting that scenario 🤔

File: src/components/MergeRequest/MergeRequestItem.tsx (hunk 1 above)
Comment (TomBerriot) [reply src/components/MergeRequest/MergeRequestItem.tsx]: @nclsHart From a UX standpoint, I think it's better to have items that signals emptyness, rather that no content, making the icons "pop" when they appear

Comment (nclsHart) [reply src/components/MergeRequest/MergeRequestItem.tsx]: The question mark say: "I don't know what you have voted", which is not the case here. 

And what about using a simple dash like in MergeRequestCI?

**Comment by nclsHart** [comment]
@TomBerriot I was thinking about miss click or omission. But you are right, it's not something we want to support.

File: src/components/MergeRequest/MergeRequestItem.tsx (hunk 1 above)
Comment (TomBerriot) [reply src/components/MergeRequest/MergeRequestItem.tsx]: Change done 👍 : 

Capture d’écran 2019-09-17 à 11 20 47

**Comment by notFloran** [comment]
@nclsHart

**Comment by nclsHart** [comment]
:tada:

**Pull request merged**